### DISCUSSION TITLE
AlphabetSearch

### DISCUSSION THREAD

**Discussion by ijc999** [EvotecIT/PSWriteHTML#348] [Q&A]
I have data that is sorted out into tables on different tabs. On one of the tables I have tried to implement Alphabet search as it has around 1000 enteries so I added it to the line starting the section New-HTMLTable.    
When I have run the code the Alphabetsearch is added to all tables on all the tabs not just the one I specified it on

**Comment by PrzemyslawKlys** [comment]
It's a bug most likely in the Alphabet plugin.
